packages/nodes-base/nodes/Google/Gmail/GmailTrigger.node.ts

```diff
@@ -284,6 +291,15 @@ export class GmailTrigger implements INodeType {
 				qs.format = 'raw';
 			}
 
+			let includeDrafts;
+			if (node.typeVersion > 1.1) {
+				includeDrafts = (qs.includeDrafts as boolean) || false;
+			} else {
+				includeDrafts = (qs.includeDrafts as boolean) || true;
```

Comment:
having || true means it will always be true, you can test it by using a node with version older than 1.2 and have include drafts object set to false, with the current setup that doesn't work
```suggestion
				includeDrafts = (qs.includeDrafts as boolean) ?? true;
```

Reply:
Thanks Shireen! Makes a lot of sense :) Nice refresher for me on Nullish Coalescing...